spyder/plugins/variableexplorer.py

```diff
@@ -114,6 +115,11 @@ def change_option(self, option_name, new_value):
             new_value = new_value[1:]
         self.sig_option_changed.emit(option_name, new_value)
 
+    @Slot()
+    def free_memory(self):
+        """Free memory signal."""
+        self.sig_free_memory.emit()

```

Comment:
Since all plugins have a reference to the wain window through `self.main`, there's no need for this. We can simply do

```python
self.main.free_memory()
```